azerothcore/azerothcore-wotlk
# fix(DB/Quest): wrong quest chain order

## Changes Proposed:
-  fix the quest chain order
- made the quest `Onu is meditating` available only after taking the quest `The Master's Glaive` that is the first 

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4269

## Tests Performed:
- tested in-game

## How to Test the Changes:
- create a night elf
- `.go cr id 3616`
- Onu should not give the quest `Onu is meditating`
- `.q add 944`
- Onu should give the quest `Onu is meditating`

## Target Branch(es):




 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
